datahub-web-react/src/app/entity/shared/containers/profile/sidebar/Ownership/AddOwnerModal.tsx

```diff
@@ -75,7 +76,7 @@ export const AddOwnerModal = ({ visible, onClose, refetch }: Props) => {
                 type: EventType.EntityActionEvent,
                 actionType: EntityActionType.UpdateOwnership,
                 entityType,
-                entityUrn: urn,
+                entityUrn: urn || '',
```

Comment:
Why would `entityUrn` be empty in Add Owner Modal? You are always adding owners to an entity. If this is for adding owners to tag as shown in the videos then should we not send tag URN in here? For all UI interactions we do want analytics. Does the analytics should up correctly?

Reply:
Fixed and committed.

[](https://github.com/linkedin/datahub/pull/4132/commits/f314d27106cd3135e7def6ac9db381a764eb4d73)

Reply:
> e a separate tag page altogether. We also need to change how the Tag Profile is rendered from the Search page! (It should still be a simple slide out. This would be inside of `Tag.tsx`.

Fixed and committed.
![edit](https://user-images.githubusercontent.com/86347578/154594091-8d8bc8c4-ad84-42d5-8e3e-11ac2d1ccc7a.png)

Reply:
> One small additional thing - Can we attempt to use an in place editing box instead of the update description modal. As shown here: https://ant.design/components/typography/ We can use the "editable" property to make the box in-place editable

Fixed and committed
![edit](https://user-images.githubusercontent.com/86347578/154594165-12d85396-1980-4065-be75-6624f4453c25.png)